### DISCUSSION TITLE
What are "H-j" and "H-k"?

### DISCUSSION THREAD

**Discussion by leira** [meow-edit/meow#303] [Q&A]
I'm trying to switch from Kakoune/Helix to Emacs+Moew. I saw this in [KEYBINDING_QWERTY.org](https://github.com/meow-edit/meow/blob/master/KEYBINDING_QWERTY.org):

```elisp
  (meow-leader-define-key
   ;; SPC j/k will run the original command in MOTION state.
   '("j" . "H-j")
   '("k" . "H-k")
```

What is "H-"? Is it "Hyper-"? What does these 2 keys do? Do they have a default binding in the hosting Emacs?

Thanks~

**Comment by snowiow** [comment]
Hyper is a modifier key from a lisp machine keyboard and no modern keyboard has this anymore. However you could rebind something you don't use to hyper. This [link](http://xahlee.info/emacs/emacs/emacs_hyper_super_keys.html) gives some ideas what could make sense to bind hyper to (f.e. fn on a mac). The other option would be to rebind these with another modifier key, so that you don't depend on the availability of a hyper key.

**Reply by leira** [reply]
Thanks for the answer!

Besides what hyper key is, what confuses me more is what `H-j` and `H-k` do? How do I find out what they do? I know `C-h k`, but I can't type `H-j` as I have no hyper key and no intention to create one.

**Reply by snowiow** [reply]
Ah in this case `H-j` does what `j` does in motion mode. Motion mode is the meow mode which is active by default in major modes like magit. To have the same navigation feel like in a normal editor session j,k is set to up and down there by meow.
But in normal emacs you have set something else on j and k in these major modes. So when you now would press `H-j` in a major mode like magit you would trigger the function which was bound to j originally.
